test/regression/expected/type_support.out

```diff
@@ -57,17 +57,17 @@ SELECT * FROM bool_tbl;
 CREATE TABLE bpchar_tbl(a CHAR(25) NOT NULL);
 INSERT INTO bpchar_tbl SELECT CAST(a AS VARCHAR) from (VALUES (''), ('test'), ('this is a long string')) t(a);
 SELECT * FROM bpchar_tbl;
-             a             
----------------------------
-                          
- test                     
- this is a long string    
+           a           
+-----------------------
+ 
+ test
+ this is a long string
```

Comment:
This change seems incorrect. I tried with `duckdb.execution = false` and that contains the trailing whitespace too. We should add the padding again when converting back to a Postgres bpchar. It seems a rather harmless difference though, so let's just create an issue for it (unless it's very simple to fix).

Reply:
I'll create issue for this.